### DISCUSSION TITLE
EVC shows under voltage alarm

### DISCUSSION THREAD

**Discussion by beaune33** [wills106/homeassistant-solax-modbus#939] [General]
From time to time my EVC shows under voltage alarm. I guess it’s not because of real undervoltage, but maybe Modbus communication problem, as the value is transferred from the inverter to the EVC by Modbus. Has anybody seen such a behaviour also? Is there any possibility to deactivate alarms? EVC firmvare version is 1.12.

**Comment by PatrikTrestik** [comment]
Hi.
Unfortunately this issue is not related to HA integration. Please contact SolaX support.
